packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -390,68 +390,29 @@ export class Summarizer extends EventEmitter implements ISummarizer {
+			lastAckRefNumber > 0
+				? lastAckRefNumber
+				: this.runtime.deltaManager.initialSequenceNumber;
 		while (this.runningSummarizer) {
 			const summaryLogger =
 				this.runningSummarizer.tryGetCorrelatedLogger(refSequenceNumber) ?? this.logger;
-			try {
-				// Initialize ack with undefined if exception happens inside of waitSummaryAck on second iteration,
-				// we record undefined, not previous handles.
-				ack = undefined;
-				ack = await this.summaryCollection.waitSummaryAck(refSequenceNumber);
-				refSequenceNumber = ack.summaryOp.referenceSequenceNumber;
-				const summaryOpHandle = ack.summaryOp.contents.handle;
-				const summaryAckHandle = ack.summaryAck.contents.handle;
-				// Make sure we block any summarizer from being executed/enqueued while
-				// executing the refreshLatestSummaryAck.
-				// https://dev.azure.com/fluidframework/internal/_workitems/edit/779
-				await this.runningSummarizer.lockedRefreshSummaryAckAction(async () =>
-					this.internalsProvider
-						.refreshLatestSummaryAck({
-							proposalHandle: summaryOpHandle,
-							ackHandle: summaryAckHandle,
-							summaryRefSeq: refSequenceNumber,
-							summaryLogger,
-						})
-						.catch(async (error) => {
-							// If the error is 404, so maybe the fetched version no longer exists on server. We just
-							// ignore this error in that case, as that means we will have another summaryAck for the
-							// latest version with which we will refresh the state. However in case of single commit
-							// summary, we might me missing a summary ack, so in that case we are still fine as the
-							// code in `submitSummary` function in container runtime, will refresh the latest state
-							// by calling `refreshLatestSummaryAckFromServer` and we will be fine.
-							if (
-								isFluidError(error) &&
-								error.errorType === DriverErrorType.fileNotFoundOrAccessDeniedError
-							) {
-								summaryLogger.sendTelemetryEvent(
-									{
-										eventName: "HandleSummaryAckErrorIgnored",
-										referenceSequenceNumber: refSequenceNumber,
-										proposalHandle: summaryOpHandle,
-										ackHandle: summaryAckHandle,
-									},
-									error,
-								);
-							} else {
-								throw error;
-							}
-						}),
-				);
-			} catch (error) {
-				summaryLogger.sendErrorEvent(
-					{
-						eventName: "HandleSummaryAckError",
-						referenceSequenceNumber: refSequenceNumber,
-						handle: ack?.summaryOp?.contents?.handle,
-						ackHandle: ack?.summaryAck?.contents?.handle,
-					},
-					error,
-				);
-			}
-			refSequenceNumber++;
+
+			summaryLogger.sendTelemetryEvent({
+				eventName: "handleSummaryAcks",
+				referenceSequenceNumber: refSequenceNumber,
+				lastAckRefNumber,
+			});
+
+			// Initialize ack with undefined if exception happens inside of waitSummaryAck on second iteration,
+			// we record undefined, not previous handles.
+			await this.summaryCollection.waitSummaryAck(refSequenceNumber);
```

Comment:
I think this await needs to move into the runningSummarizer.handleSummaryAck() function. That function assumes that an ack has already been awaited for and if that has not happened, it will end up returning -1 and this logic will break.